NutritionManagement/src/io/OutputFormatter.java

```diff
@@ -6,10 +6,14 @@ public class OutputFormatter {
     public static void printTotalNutritionValue(NutritionValue nutritionValue) {
         System.out.println("Total Nutrition Value:");
         System.out.println("----------------------");
-        System.out.println("Calories: " + formatDouble(nutritionValue.getCalories()) + " kcal");
-        System.out.println("Protein: " + formatDouble(nutritionValue.getProtein()) + " g");
-        System.out.println("Carbs: " + formatDouble(nutritionValue.getCarbs()) + " g");
-        System.out.println("Fat: " + formatDouble(nutritionValue.getFat()) + " g");
+        System.out.println("Calories: " + formatDouble(nutritionValue.getCalories()) + " kcal, "
```

Comment:
Provide more informative output by including the percentage representation,  help users make better dietary decisions.